DemoServer/AdditionalLanguages/nodejs/demo/queries/projectingUsingFunctions/index.js

```diff
@@ -0,0 +1,32 @@
+//region Usings
+const { documentStore } = require('../../common/docStoreHolder');
+//endregion
+async function run () {
+    //region Demo
+    const session = documentStore.openSession();
+
+    //region Step_1
+    const rawQueryString = `declare function output(employee) {
+            var formatTitle = function(employee) { return "Title: " + employee.Title };
+            var formatName  = function(employee) { return "FullName: " + employee.FirstName + " " + employee.LastName; };
+            return { Title : formatTitle(employee), FullName : formatName(employee) };
+        }
+    //endregion
+    //region Step_2
+        from Employees as employee select output(employee)`;
+    //endregion
+
+    //region Step_3
+    const projectedQueryWithFunctions  = await session.advanced.rawQuery(rawQueryString);
```

Comment:
I don't think we can await on this. (we should await on `all` call below)

Reply:
also please remove double spaces around `=`

Reply:
Done